creator-node/src/services/stateMachineManager/stateMonitoring/findReplicaSetUpdates.jobProcessor.ts

```diff
@@ -53,6 +54,11 @@ async function findReplicaSetUpdates({
   DecoratedJobReturnValue<FindReplicaSetUpdatesJobReturnValue>
 > {
   const unhealthyPeersSet = new Set(unhealthyPeers || [])
+  const contentNodeInfoManager = ContentNodeInfoManager(logger)
+  const cNodeEndpointToSpIdMap =
+    await contentNodeInfoManager.getMapOfCNodeEndpointToSpId()
+  const cNodeEndpointToSpIdMapString =
+    await contentNodeInfoManager.getSerializedMapOfCNodeEndpointToSpId()
```

Comment:
this would fetch same info from redis twice unnecessarily - can we improve this pattern a bit to only make one network call? few diff approaches i can think of - either add function to serialize a typed param, or cache value in memory (this seems like a step backward imo) since we prob want to keep the ContentNodeInfoManager fully stateless right

Reply:
improved to just use a util to stringify the Map